lib/glific/clients/common_webhook.ex

```diff
@@ -201,11 +203,17 @@ defmodule Glific.Clients.CommonWebhook do
            Bhasini.with_config_request(
              fields,
              contact.language.locale
-           ) do
-      params = Jason.decode!(response.body)
+           ),
+         %{"feedbackUrl" => _feedback_url, "pipelineInferenceAPIEndPoint" => _endpoint} = params <-
+           Jason.decode!(response.body) do
       Glific.Bhasini.text_to_speech(params, text, org_id)
     else
-      _ -> %{success: false, reason: "GCS is disabled or something went wrong with Bhasini"}
+      true ->
+        %{success: false, reason: "GCS is disabled"}
+
+      error ->
+        Logger.error("Error received from Bhasini: #{error["message"]}")
```

Comment:
`Bhasini.with_config_request` returns `{:error, %{succes => boolean, msg => String.t()}}`, but here we trying to access "message", which will cause error